src/commands/jobs/pause.ts

```diff
@@ -1,19 +1,59 @@
 import { runCommand } from '../../util/command';
-import { detailOutputFormat } from './util';
+import { detailOutputFormat, jobsSearchArgumentsDefault, searchCriteriaIsMissing, getSearchArgs } from './util';
 
 export = {
-	command: 'pause <id>',
-	desc: 'Pause job',
+	command: 'pause [id]',
+	desc: 'Pause job(s)',
+	builder: yargs => {
+		yargs.options({
+			'all': {
+				desc: 'Pause all jobs',
+				type: 'boolean',
+				group: 'Search',
+				default: undefined
+			},
+			...jobsSearchArgumentsDefault
+		})
+	},
 	handler: argv => {
-		runCommand(argv, async (client, output) => {
-			const job = await client.jobs.pause(argv.id, {
-				fields: [
-					'name',
-					'status',
-					'disabled'
-				]
+		if (argv.id === undefined) {
+			if (searchCriteriaIsMissing(argv)) {
+				console.error("Search criteria must be specified. To pause all jobs, use --all parameter");
```

Comment:
I would output this as red text similar to what we do in `writeFailure` [here](https://github.com/portalarchitects/skysync-cli/blob/master/src/util/formatter.ts#L47)